views/js/runner/proxy.js

```diff
@@ -173,11 +175,16 @@ define([
             },
 
             /**
-             * Install the proxy. Optionnal.
+             * Install the proxy.
              * This step let's attach some features before the proxy reallys starts (before init).
+             *
+             * @param {Map} holder - the test runner data holder
              * @returns {*}
              */
-            install: function install() {
+            install: function install(dataHolder) {
```

Comment:
Just an idea, could this belong to a factory call instead of creating a new lifecycle step whose purpose can be confusing? (install vs init)

Reply:
No, this can't be done within the factories since the goal is to delegate to the provider. We already use the install step, with the same lifecycle in other places (ie. plugins, proxy, tr, etc.). Confusion if there is, can be compensated by documentation, or to rename `init` to something more meaningful like `mount` ?